### DISCUSSION TITLE
cp _list

### DISCUSSION THREAD

**Discussion by iAmMrNegative** [AdriaGual/marvel-snap-bot#38] [General]
@AdriaGual My cp_list has all of the cards that are currently out i wish i would have done them in pool order but they are mostly in A-Z 
[cp_list.zip](https://github.com/AdriaGual/marvel-snap-bot/files/10553431/cp_list.zip)

**Comment by iAmMrNegative** [comment]
if you do use this might have to go through and add a _ between names i did all solid one names.

doctor_doom = doctordoom

**Comment by AdriaGual** [comment]
OMG, that's an impressive work!! Huge thanks, you can do a PR so me and other people can review it!🚀
